src/components/vega/VegaPlot.js

```diff
@@ -0,0 +1,47 @@
+import React, { Suspense, useMemo } from 'react';
+import { Handler } from 'vega-tooltip';
+
+const ReactVega = React.lazy(() => import('./ReactVega'));
+
+const DATASET_NAME = 'table';
```

Comment:
Could this change in the future?

Reply:
For some more complex vega plots there may need to be multiple named datasets https://vega.github.io/vega-lite/docs/data.html#named

but for most vega plots one should work. I think if we find that more than one dataset is needed in the future then this can be extracted out of this component.

Reply:
Ok, I see what this is now.  Sounds good.